.github/workflows/benchmark_nightly_cpu.yml

```
@@ -0,0 +1,25 @@
+name: Benchmark torchserve cpu nightly
+
+on:
+  # run every day at 2:15am
+  schedule:
+    - cron:  '15 02 * * *'
+jobs:
+  nightly:
+    runs-on: self-hosted
```

I added tags to github runner otherwise each CI job will pick the first available one

```suggestion
    runs-on: [self-hosted, cpu]
```